src/Command/ContaoFrameworkDependentInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+namespace Contao\CoreBundle\Command;
+
+/**
+ * Interface FrameworkDependentCommandInterface
+ * Only boot Contao-Framework if Command implements this Interface
+ *
+ * @author Dominik Tomasi <https://github.com/dtomasi>
+ */
+interface ContaoFrameworkDependentInterface {
```

Comment:
there should be a line break here and a line break as last line of the file to follow PSR